.changeset/blue-lions-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `Segmentation fault` errors
```

Comment:
Sorry for trivial things, can you please remove backticks since it's not a code?

```suggestion
Fixed: Segmentation fault errors
```

Reply:
Ah, touching "Cosmiconfig" may be better:

```suggestion
Fixed: Segmentation fault errors for Cosmiconfig 8.2
```

Reply:
Thank you for still paying attention to these things 🙇